sdk/canton/community/bindings-java/src/test/scala/com/daml/ledger/javaapi/data/PackageVersionSpec.scala

```diff
@@ -0,0 +1,40 @@
+package com.daml.ledger.javaapi.data
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import scala.util.Random
+
+class PackageVersionSpec extends AnyFlatSpec with Matchers {
```

Comment:
Not sure why this fail is here. It's unrelated to the other changes in this PR I think